packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -1,9634 +1,9644 @@
-    // (undocumented)
-    cyan = 23,
-    // (undocumented)
-    darkBlue = 2,
-    // (undocumented)
-    darkGreen = 6,
-    // (undocumented)
-    darkRed = 14,
-    // (undocumented)
-    gold = 18,
-    gray = 22,
-    // (undocumented)
-    green = 5,
-    // (undocumented)
-    lightBlue = 0,
-    // (undocumented)
-    lightGreen = 4,
-    // (undocumented)
-    lightPink = 7,
-    // (undocumented)
-    lightRed = 17,
-    // (undocumented)
-    magenta = 9,
-    // (undocumented)
-    orange = 12,
-    // (undocumented)
-    pink = 8,
-    // (undocumented)
-    purple = 10,
-    // @deprecated
-    red = 13,
-    // (undocumented)
-    rust = 24,
-    // (undocumented)
-    teal = 3,
-    transparent = 15,
-    // (undocumented)
-    violet = 16,
-    // (undocumented)
-    warmGray = 20
-}
-
-// @public (undocumented)
-export enum PersonaPresence {
-    // (undocumented)
-    away = 3,
-    // (undocumented)
-    blocked = 5,
-    // (undocumented)
-    busy = 6,
-    // (undocumented)
-    dnd = 4,
-    // (undocumented)
-    none = 0,
-    // (undocumented)
-    offline = 1,
-    // (undocumented)
-    online = 2
-}
-
-// @public (undocumented)
-export namespace personaPresenceSize {
-    const // (undocumented)
-    size6 = "6px";
-    const // (undocumented)
-    size8 = "8px";
-    const // (undocumented)
-    size12 = "12px";
-    const // (undocumented)
-    size16 = "16px";
-    const // (undocumented)
-    size20 = "20px";
-    const // (undocumented)
-    size28 = "28px";
-    const // (undocumented)
-    size32 = "32px";
-    const // @deprecated (undocumented)
-    border = "2px";
-}
-
-// @public (undocumented)
-export enum PersonaSize {
-    // @deprecated
-    extraExtraSmall = 1,
-    // @deprecated
-    extraLarge = 6,
-    // @deprecated
-    extraSmall = 2,
-    // @deprecated
-    large = 5,
-    // @deprecated
-    regular = 4,
-    // @deprecated
-    size10 = 9,
-    size100 = 15,
-    size120 = 18,
-    // @deprecated
-    size16 = 8,
-    size24 = 10,
-    // @deprecated
-    size28 = 7,
-    size32 = 11,
-    size40 = 12,
-    size48 = 13,
-    size56 = 16,
-    size72 = 14,
-    size8 = 17,
-    // @deprecated
-    small = 3,
-    // @deprecated
-    tiny = 0
-}
-
-// @public (undocumented)
-export namespace personaSize {
-    const // (undocumented)
-    size8 = "20px";
-    const // (undocumented)
-    size10 = "20px";
-    const // (undocumented)
-    size16 = "16px";
-    const // (undocumented)
-    size24 = "24px";
-    const // (undocumented)
-    size28 = "28px";
-    const // (undocumented)
-    size32 = "32px";
-    const // (undocumented)
-    size40 = "40px";
-    const // (undocumented)
-    size48 = "48px";
-    const // (undocumented)
-    size56 = "56px";
-    const // (undocumented)
-    size72 = "72px";
-    const // (undocumented)
-    size100 = "100px";
-    const // (undocumented)
-    size120 = "120px";
-}
-
-// @public
-export const Pivot: React.FunctionComponent<IPivotPro
```

Comment:
Why is this file changing like this?

Reply:
No idea. It's done that after a full build and merge with master. I'll try doing it again once I get tests going.